app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
@tijmenb No. The `%w(published unpublished)` (note - not `withdrawn`) filter will return more content_ids than just those exposed by the live content store, because it will return ids for all types of unpublishings. See the link Graham provided for the other types: https://github.com/alphagov/publishing-api/blob/master/app/models/unpublishing.rb#L7-L10